<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Giduba</string>
        </property>
        <widget class="QMenuBar" name="menu">
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>&amp;File</string>
                </property>
                <action name="actionNew">
                    <property name="text">
                        <string>New</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+N</string>
                    </property>
                </action>
                <action name="actionOpen">
                    <property name="text">
                        <string>Open</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+O</string>
                    </property>
                </action>
                <action name="actionSave">
                    <property name="text">
                        <string>Save</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+S</string>
                    </property>
                </action>
                <action name="actionSaveAs">
                    <property name="text">
                        <string>Save As</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+Shift+S</string>
                    </property>
                </action>
                <action name="actionExit">
                    <property name="text">
                        <string>Quit</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+Q</string>
                    </property>
                </action>
                <addaction name="actionNew"/>
                <addaction name="separator"/>
                <addaction name="actionOpen"/>
                <addaction name="separator"/>
                <addaction name="actionSave"/>
                <addaction name="actionSaveAs"/>
                <addaction name="separator"/>
                <addaction name="actionExit"/>
            </widget>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>&amp;Edit</string>
                </property>
                <action name="actionUndo">
                    <property name="text">
                        <string>Undo</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+Z</string>
                    </property>
                </action>
                <action name="actionRedo">
                    <property name="text">
                        <string>Redo</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+Shift+Z</string>
                    </property>
                </action>
                <action name="actionSelectAll">
                    <property name="text">
                        <string>Select All</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+A</string>
                    </property>
                </action>
                <action name="actionToggleSearch">
                    <property name="text">
                        <string>Toggle Search/Replace</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+H</string>
                    </property>
                </action>
                <action name="actionSearch">
                    <property name="text">
                        <string>Search</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+F</string>
                    </property>
                </action>
                <action name="actionReplace">
                    <property name="text">
                        <string>Replace</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+R</string>
                    </property>
                </action>
                <action name="actionCut">
                    <property name="text">
                        <string>Cut</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+X</string>
                    </property>
                </action>
                <action name="actionCopy">
                    <property name="text">
                        <string>Copy</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+C</string>
                    </property>
                </action>
                <action name="actionPaste">
                    <property name="text">
                        <string>Paste</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+V</string>
                    </property>
                </action>
                <action name="actionLineWrap">
                    <property name="text">
                        <string>Line Wrap</string>
                    </property>
                    <property name="shortcut">
                        <string>Ctrl+L</string>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>true</bool>
                    </property>
                </action>
                <addaction name="actionUndo"/>
                <addaction name="actionRedo"/>
                <addaction name="separator"/>
                <addaction name="actionToggleSearch"/>
                <addaction name="actionSearch"/>
                <addaction name="actionReplace"/>
                <addaction name="actionSelectAll"/>
                <addaction name="separator"/>
                <addaction name="actionCut"/>
                <addaction name="actionCopy"/>
                <addaction name="actionPaste"/>
                <addaction name="separator"/>
                <addaction name="actionLineWrap"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>&amp;Help</string>
                </property>
                <action name="actionAbout">
                    <property name="text">
                        <string>About</string>
                    </property>
                    <property name="shortcut">
                        <string>F1</string>
                    </property>
                </action>
                <addaction name="actionAbout"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuEdit"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QWidget" name="main">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QTextEdit" name="editor"/>
                </item>
                <item>
                    <widget class ="QWidget" name="searchReplaceElement">
                        <property name="visible">
                            <bool>false</bool>
                        </property>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="searchLabel">
                                    <property name="text">
                                        <string>Search:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="searchBox"/>
                            </item>
                            <item>
                                <widget class="QPushButton" name="searchButton">
                                    <property name="text">
                                        <string>Search</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLabel" name="replaceLabel">
                                    <property name="text">
                                        <string>Replace:</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="replaceBox"/>
                            </item>
                            <item>
                                <widget class="QPushButton" name="replaceButton">
                                    <property name="text">
                                        <string>Replace</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QStatusBar" name="status"/>
                </item>
            </layout>
        </widget>

    </widget>
    <resources/>
    <connections/>
</ui>
